CMakeLists.txt

```diff
@@ -161,32 +163,34 @@ ELSE (GIT_BINARY)
 	ENDIF (NOT GIT_FOUND)
 ENDIF (GIT_BINARY)
 
-IF (PREFIX)
+# PREFIX has no sence in case of win32
+IF (PREFIX AND NOT WIN32)
 	ADD_DEFINITIONS(-DPREFIX=${PREFIX})
-ENDIF (PREFIX)
+ENDIF (PREFIX AND NOT WIN32)
 
 # Set build types and display info
-	MESSAGE(STATUS "DYNAMIC_LINKING               : ${DYNAMIC_LINKING}\n")
 IF(RELEASE)
 	MESSAGE(STATUS "CMAKE_INSTALL_PREFIX          : ${CMAKE_INSTALL_PREFIX}")
 	MESSAGE(STATUS "BIN_PREFIX                    : ${BIN_PREFIX}")
 	MESSAGE(STATUS "DATA_PREFIX                   : ${DATA_PREFIX}")
+    IF(LOCALIZE)
+        MESSAGE(STATUS "LOCALE_PATH                   : ${LOCALE_DIR}")
+    ENDIF(LOCALIZE)
 	MESSAGE(STATUS "DESKTOP_ENTRY_PATH            : ${DESKTOP_ENTRY_PATH}")
 	MESSAGE(STATUS "PIXMAPS_ENTRY_PATH            : ${PIXMAPS_ENTRY_PATH}")
 	MESSAGE(STATUS "PIXMAPS_UNITY_ENTRY_PATH      : ${PIXMAPS_UNITY_ENTRY_PATH}")
 	MESSAGE(STATUS "MANPAGE_ENTRY_PATH            : ${MANPAGE_ENTRY_PATH}\n")
 	SET(CMAKE_BUILD_TYPE RELEASE)
+	ADD_DEFINITIONS(-DRELEASE)
 	# FIXME: Removed -Werror for a while. Currently it is broken in upstream.
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Os -std=c++11 -MMD -Wall -Wextra")
 	IF(MINGW)
 		# Workaround for: http://ehc.ac/p/mingw/bugs/2250/
 		ADD_DEFINITIONS(-D__NO_INLINE__)
 	ENDIF(MINGW)
-	IF(NOT WIN32 AND NOT PREFIX)
-		# PREFIX has no sence in case of win32
-		ADD_DEFINITIONS(-DDATA_DIR_PREFIX=${CMAKE_INSTALL_PREFIX})
-	ENDIF(NOT WIN32 AND NOT PREFIX)
-	ADD_DEFINITIONS(-DRELEASE)
+#	IF(NOT WIN32 AND NOT PREFIX)
```

Comment:
Why is it commented out?

In cata_tiles.cpp:
```C++
    std::string img_path = path;
#ifdef PREFIX   // use the PREFIX path over the current directory
    img_path = (FILENAMES["base_path"] + "/" + img_path);
#elif defined DATA_DIR_PREFIX // Used for linux release installs
    img_path = (FILENAMES["datadir"] + "/" + img_path);
#endif
```
And in path_info.cpp:
```C++
#ifdef DATA_DIR_PREFIX
        update_pathname("datadir", FILENAMES["base_path"] + "share/cataclysm-dda/");
        update_pathname("gfxdir", FILENAMES["datadir"] + "gfx/");
        update_pathname("luadir", FILENAMES["datadir"] + "lua/");
#else
        update_pathname("datadir", FILENAMES["base_path"] + "data/");
        update_pathname("gfxdir", FILENAMES["base_path"] + "gfx/");
        update_pathname("luadir", FILENAMES["base_path"] +"lua/");
#endif
```

This would lead to datadir being "[base_path]/share/cataclysm-dda/" even on windows, the tileset images (cata_tiles.cpp) would be loaded from there, is that correct? And if that's all fine, why not remove the `IF` completely instead of just commenting it?

PS: I see, the strange "sence" comes from here.